src/database/models/user.js

```diff
@@ -9,6 +9,10 @@ export default (sequelize, DataTypes) => {
      */
     static associate(models) {
       // define association here
+      User.hasOne(models.Role, {
+        foreignKey: 'id',
+        onDelete: 'CASCADE',
+      });
```

Comment:
good job here, just revert to es6 though

Reply:
> good job here, just revert to es6 though

@itisWasp you seem to have missed this